### ISSUE TITLE
Game parameter isn't used

### ISSUE THREAD

**Issue by drbirbdev** [Vodianoi/uploadMod#1]
When uploading to nexus, game parameter is ignored, causing upload to always be for valheim (I think the default in opmu)

**Comment by Vodianoi** [comment]
Yep sorry this not a finished work, the `opmu` command always return rest http error in js but works using `run` command directly in the action. Need to figure out why..
I removed the `-g` for testing purpose but it is not the source of the issue. 
See this:
https://github.com/Digitalroot-Valheim/Digitalroot.OdinPlusModUploader/issues/10

I will update this repo as release 1.0.0 once it is working and update the readme to avoid confusion.

**Comment by Vodianoi** [comment]
Should be fixed now : [0.9.40](https://github.com/Vodianoi/uploadMod/releases/tag/0.9.40)
Didn't test for other games, please report any issue !

**Comment by drbirbdev** [comment]
Looks like it was able to discover the game.  I was able to manually tweak some of the other arguments and get a file to upload for Stardew Valley.

https://github.com/drbirbdev/StardewValley/actions/runs/3798088712/jobs/6459540254

version 1.0.3 was uploaded with the github action https://www.nexusmods.com/stardewvalley/mods/14170?tab=files

The run did have some errors

```/usr/bin/cp tcli
/usr/bin/cp: missing destination file operand after 'tcli'
Try '/usr/bin/cp --help' for more information.
Error: Error: The process '/usr/bin/cp' failed with exit code 1
Error: Error: ENOENT: no such file or directory, chdir '/home/runner/work/StardewValley/StardewValley' -> ''
```

which I assume is because I've ignored requirements for thunderstore.

I'd also have an additional request to explicitly quote archiveFile, the same way you do description.  Stardew modding API defaults to "modname semver" as a zip file, and this action can't handle the space as-is.  I worked around that by manually renaming the zip file for testing.

Altogether though, this works great.  It's going to make releasing updates a lot easier, so thank you for this.
